pkgs/servers/teleport/default.nix

```diff
@@ -63,7 +94,7 @@ buildGoModule rec {
   '';
 
   meta = with lib; {
-    description = "A SSH CA management suite";
+    description = "Certificate authority and access plane for SSH, Kubernetes, web applications, and databases";
```

Comment:
Teleport now does much more than just manage SSH certificates. Took this description straight from the GitHub repo, hopefully it's not too marketing-y.